assets/js/src/public.jsx

```diff
@@ -81,6 +81,24 @@ jQuery(($) => {
     }
   }
 
+  function displaySuccessMessage(form) {
+    // hide all form elements
+    form.children().each((index, element) => {
```

Comment:
You can use jquery's [.not selector](https://api.jquery.com/not/) to exclude '.mailpoet_message' and get rid of the forEach. I think it would be a bit nicer to read.